jsonrpc/src/content/results.json

```diff
@@ -99,10 +99,9 @@
   },
   "OfferResult": {
     "name": "offerResult",
-    "description": "Returns the number of peers that the content was gossiped to",
+    "description": "Returns \"true\" upon success",
```

Comment:
Does it make sense to return the accepted response, instead of a boolean? I don't have a strong opinion, maybe returning a boolean is more reasonable because we don't care for the response, we only want to make sure the request was sent.

Reply:
Fluffy has a version of this RPC (albeit named differently for now), which returns true if:
- Accept response received but with no bits set for the content keys (so the receiver doesn't want any of the data but did reply)
- Accept response with certain content accepted + that content got successfully send over uTP (as far as we can see from our side). 

It throws an error if:
- No accept response received
- Data transmission failures (uTP)
- Invalid content keys bitlist.

So the request does currently wait for a response before returning.

I think we could alter the boolean perhaps to:
- Return the actual bitlist / content key list if it got successfully send
- Return the empty bitlist if no data needed to be send
- Return error in any case of the error cases.

This would be close to returning the actual `Accept` response, minus the `connectionId` which is of no real use unless you want to debug / initiate the uTP part manually.

Reply:
> I think we could alter the boolean perhaps to:
> 
> * Return the actual bitlist / content key list if it got successfully send
> * Return the empty bitlist if no data needed to be send
> * Return error in any case of the error cases.

This seems reasonable to me :+1: